### ISSUE TITLE
Forefoot abductus

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11175]
**Preferred term label:**
Forefoot abductus

**Synonyms**
Forefoot abduction

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C0410776

On X-ray:
AP foot x-ray showing increased Simmon angle (angle between the talus and first metatarsal, normal is 7° ± 4–5°) or increased talonavicular uncoverage of >30-40%

Clinically:
"too many toes" sign; malalignment of the foot causing the forefoot to abduct, which can be visualized by standing behind the patient and seeing more than the normal number of toes visible on the lateral aspect of the foot

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Posterior tibial tendon insufficiency (PTTI) (https://www.physio-pedia.com/Tibialis_posterior_rupture)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001397
- label: Forefoot abduction
- parents: [Abnormal foot morphology (HP:0001760)](https://hpo.jax.org/app/browse/term/HP:0001760)
- pmid: PMID:31194317
- def: Outward deviation or turning of the front part of the foot away from the body's midline. An anteroposterior foot x-ray shows increased Simmon angle (angle between the talus and first metatarsal, normal is 7 degrees plus/minus 4 to 5 degrees) or increased talonavicular uncoverage of over 30 to 40 percent. A common clinical observation is the too many toes sign; when viewed from behind, more than two toes will flare out laterally due to the valgus alignment of the foot.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11175
- synonyms:
